apps/mobile/src/store/settings/settings.ts

```diff
@@ -67,7 +67,7 @@ export const initialState: SettingsState = {
   fiatCurrencyPreference: 'USD',
   networkPreference: WalletDefaultNetworkConfigurationIds.mainnet,
   privacyModePreference: 'visible',
-  securityLevelPreference: 'insecure',
+  securityLevelPreference: 'undefined',
```

Comment:
I think this is going to break app auth in settings, why does this need to be `undefined`? Can't we just change the code you initially wrote to handle a default of `insecure`?

Reply:
If not, please look at the app auth settings and make sure to cover `undefined` there.

Reply:
In the app auth setting, we use a toggle so rn it is just handling 2 states. When a user gets to that point it should be defined though as `secure` or `insecure`? So, maybe ok?

Reply:
Thanks for addressing, lgtm!